lib/libc/include/loongarch-linux-any/asm/bitsperlong.h

```diff
@@ -1,9 +0,0 @@
-/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
```

Comment:
I need to verify that this removal is actually ok. It looks a bit suspect.

Reply:
```console
❯ cat bitsperlong.c
#include <linux/unistd.h>
#include <stdio.h>
int main() { printf("%d\n", __BITS_PER_LONG); }
❯ zig cc linux.c -target loongarch64-linux
LLVM ERROR: Cannot select: 0x7579756fb7f0: i64 = fp_to_fp16 0x7579752aaf60
  0x7579752aaf60: f32,ch = CopyFromReg 0x7579743403f0, Register:f32 %11
    0x7579754de450: f32 = Register %11
In function: __fixhfsi
fish: Job 1, '../zig/build/debug/bin/zig cc l…' terminated by signal SIGABRT (Abort)
```

(The LLVM error is unrelated and will go away with 19 - the important thing is that `__BITS_PER_LONG` is defined as expected.)